### PR TITLE
Clickable rows, only create link if the decorator says this model should...

### PR DESCRIPTION
**Pull request by TracyMB**
... have a clickable row

### CODE REVIEW

File: app/views/nunchaku/resources/_table_row.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <tr id="<%= resource_class.name.underscore %>_<%= resource.id %>" class="<%= resource.tints %>">
 	<% column_names.each do |column| %>
-	  <td class="cell <%= column %> <%= "clickable" if resource.try(:clickable_row?) %>" resource-link = "<%= resource_path(resource) %>">
+	  <td class=" <%= cell_classes(column) %>" resource-link = "<%= resource_cell_link(resource) %>">
```
Comment (MarkRatjens) [inline app/views/nunchaku/resources/_table_row.html.erb]: this could not be done at the tr level?

Comment (TracyMB) [inline app/views/nunchaku/resources/_table_row.html.erb]: I tried it first on the tr level, but I needed to exclude the 'edit' cells in the table, so I changed to the cell level.

Comment (john-song-marketboomer) [inline app/views/nunchaku/resources/_table_row.html.erb]: Tracy is going to move it to tr level in another PR, this PR will be merged to allow support guys to test stuff on staging
